src/node/chainstatemanager_args.cpp

```diff
@@ -60,8 +60,7 @@ util::Result<void> ApplyArgsManOptions(const ArgsManager& args, ChainstateManage
         script_threads += GetNumCores();
     }
     // Subtract 1 because the main thread counts towards the par threads.
-    opts.worker_threads_num = std::clamp(script_threads - 1, 0, MAX_SCRIPTCHECK_THREADS);
-    LogPrintf("Script verification uses %d additional threads\n", opts.worker_threads_num);
```

Comment:
The max number of workers threads before was 15 and now seems to be 14. E.g. provide `-par=16`:

Previous Code: 
```
script_threads = 16.
worker_threads_num = std::clamp(script_threads-1, 0, 15);
...
Outcome: 15 workers
```

Current Code
```
script_threads = std::clamp(16, 0, 15); -> output 15
worker_threads_num = script_threads - 1;
..
Outcome: 14 workers.
```

Reply:
I'm confused, this is printing `Script verification uses 15 additional threads` for both master and this commit when I provide `-par=16`. As far as I can tell the logic as applied is:
```
worker_threads_num = script_threads - 1;
worker_threads_num = std::clamp(worker_threads_num, 0 , 15);
```
Which should be equivalent?

Reply:
Yeah, never mind. The subtraction happens before the clamping, not after as I wrote above. So the behavior remains the same.
I guess I'm a bit sleepy today.. sorry.